stable/ksoc-plugins/Chart.yaml

```diff
@@ -17,7 +17,7 @@ annotations:
   # Possible kind options are added, changed, deprecated, removed, fixed and security.
   artifacthub.io/changes: |
     - kind: changed
-      description: documentation
+      description: dependencies
```

Comment:
What does this do?  `dependencies` is not listed as an option in the comment above

Reply:
It will just state that we've changed the dependencies of the chart.

Reply:
I think the list refers to operations for `kind`. `description` just says what was operated on